_search-plugins/search-pipelines/neural-query-enricher.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Neural query enrich
```

Comment:
Should title mirror the wording used in line 10?

Reply:
Since this page is under the "Search processors" heading in the left nav, I'm just omitting "processor" here so the string is shorter. I do the same with the other processors.